README.md

```diff
@@ -1,135 +1,450 @@
+     throw new Error(`Expected ${requiredAccounts} accounts in devnet-accounts.json, found ${devnetAccounts.accounts.length}`);
+   };
+   ```
+
+4. Copy and paste the code to configure network information from the `solidity/hardhat.config.js` file into the `hardhat.config.js` file for your project. 
+
+   For example:
+
+   ```javascript
+     defaultNetwork: "kadena_devnet0",
+     networks: {
+       kadena_devnet0: {
+         url: 'http://localhost:8545',
+         chainId: 1789,
+         accounts: devnetAccounts.accounts.map(account => account.privateKey),
+         chainwebChainId: 0,
+       },
+       kadena_devnet1: {
+         // url: 'http://localhost:8546',
+         url: 'http://localhost:8555',   
+         chainId: 1790,
+         accounts: devnetAccounts.accounts.map(account => account.privateKey),
+         chainwebChainId: 1,
+       },
+     },
+     ```
+
+1. (Optional) Copy and paste the code to configure etherscan settings from the `solidity/hardhat.config.js` file into the `hardhat.config.js` file for your project.
+   
+   For example:
+
+   ```javascript
+     defaultNetwork: "kadena_devnet0",
+     etherscan: {
+       apiKey: {
+         'kadena_devnet0': 'empty',
+         'kadena_devnet1': 'empty',
+       }, 
+       customChains: [
+         {
+           network: "kadena_devnet0",
+           chainId: 1789,
+           urls: {
+             apiURL: "http://localhost:8000/api",
+             browserURL: "http://localhost:8000"
+           }
+         },
+         {   
+           network: "kadena_devnet1",
+           chainId: 1790,
+           urls: {
+             apiURL: "http://localhost:8001/api",
+             browserURL: "http://localhost:8001"
+           }
+         },
+       ]
+     },
+     ```
+1. Save your changes and close the `hardhat.config.js` file.
+
+### Compiling and testing integration
+
+After configuring your Hardhat project to use the Chainweb EVM development network, you can compile, test, and deploy the project using standard `hardhat` commands.
+For example, compile the project by running the following command:
 
 ```sh
-./network devnet pull
+npx hardhat compile
 ```
 
-Start the network:
+Run unit tests using the standard `hardhat test` command:
 
 ```sh
-./network devnet start
+cd solidity 
+npx hardhat test
 ```
 
-Check that blocks are being produced:
+Similarly, if you have a `deploy.js` deployment script in a `scripts` directory, you can deploy the project using the standard `hardhat` command to run scripts:
 
 ```sh
-./network devnet status
+npx hardhat run scripts/deploy.js
 ```
 
-Call the previous command repeatedly and check that the cut height value is
-increasing.
+### Changing the default network
 
-Explore available commands for managing and monitoring the network:
+If you prefer to keep the `hardhat` network as the default network for your Hardhat project, you can configure `kadena_devnet0` as a separate network.
```

Comment:
This isn't completely correct. Any network that is not the internal hardhat network must be configured as a network in hardhat.config.js. Even if you specify `kadena_devnet0` as the defaultNetwork, it still must be configured. Otherwise, hardhat doesn't know what `kadena_devnet0` is.

I pushed a sommit with the `defaultNetwork: "kadena_devnet0",` uncommented. I would say here:

If you prefer to keep the `hardhat` network as the default network for your Hardhat project, comment out or remove `defaultNetwork: "kadena_devnet0"`from solidity/hardhat.config.js.